nix/ec2-properties.nix

```diff
@@ -1,64 +1,97 @@
 {
   "c1.medium" = { cores = 2; memory = 1740; allowsEbsOptimized = false; };
-  "c1.xlarge" = { cores = 8; memory = 7168; allowsEbsOptimized = true; };
-  "c3.2xlarge" = { cores = 8; memory = 15360; allowsEbsOptimized = true; };
-  "c3.4xlarge" = { cores = 16; memory = 30720; allowsEbsOptimized = true; };
+  "c1.xlarge" = { cores = 8; memory = 7168; allowsEbsOptimized = false; };
```

Comment:
I’m not quite sure why this has been changed. Is there a bug in `nix/generate-ec2-properties.py`?

Reply:
It looks like some of the newer instance types that have EBS optimization on by default [1] do not really support disabling it, basically making disabling it a no-op. Not sure why the c3's changed though. 

[1] https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSOptimized.html

Reply:
I will review generate-ec2-properties.py today, and check if I can see an obvious bug.

Reply:
Looking at the JSON that AWS produces, it seems that they do not provide the ebsOptimized flag anymore in the old generation models in the pricing JSON file.